docs/guide/rsun.rst

```diff
@@ -0,0 +1,69 @@
+.. _sunpy-topic-guide-rsun:
```

Comment:
```suggestion
.. _sunpy-topic-guide-role-of-rsun:
```
I made some of them quite verbose in my PR and I think that would be nice here as well :P